homeassistant/components/subaru/sensor.py

```diff
@@ -271,14 +266,9 @@ def available(self) -> bool:
             return False
         return last_update_success
 
-    def get_current_value(self):
+    def get_current_value(self) -> None | int | float:
```

Comment:
I think we can inline this function in `native_value` as that's the only place it's used.